superset-frontend/plugins/plugin-chart-table/src/utils/extent.ts

```diff
@@ -24,16 +24,16 @@ export default function extent<T = number | string | Date | undefined | null>(
   // eslint-disable-next-line no-restricted-syntax
   for (const value of values) {
     if (value !== null) {
-      if (min === undefined) {
+      if (min === undefined || min === null) {
```

Comment:
Maybe we could use `_.isNil` here?

```node
let _ = require('lodash');
> _.isNil(undefined)
true
> _.isNil(null)
true
> _.isNil(0)
false
> _.isNil('')
false
```

Reply:
Done in https://github.com/apache/superset/pull/24272/commits/cbcc0eead2226cbd31a0627dcfb09bba0c1e42d6